Added paste from word icon to tinymce in test-center
Index: test-center/ambrosia/ambrosia-library/lib/src/webapp/js/ambrosia_1-1-0.js
===================================================================
--- test-center/ambrosia/ambrosia-library/lib/src/webapp/js/ambrosia_1-1-0.js	(revision 51568)
+++ test-center/ambrosia/ambrosia-library/lib/src/webapp/js/ambrosia_1-1-0.js	(working copy)
@@ -855,9 +855,9 @@
 				mode: tinyMode,
 				editor_selector: "ambrosiaHtmlEdit_full",
 				convert_urls: false,
-				plugins: "safari,fullscreen",
+				plugins: "safari,fullscreen,paste",
 				theme: "advanced",
-				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 				theme_advanced_buttons2: "",
 				theme_advanced_buttons3: "",
 				extended_valid_elements: "+a[id|style|rel|rev|charset|hreflang|dir|lang|tabindex|accesskey|type|name|href|target:_blank|title|class|onfocus|onblur|onclick|" + 
@@ -881,9 +881,9 @@
 				mode: tinyMode,
 				editor_selector: "ambrosiaHtmlEdit_tall",
 				convert_urls: false,
-				plugins: "safari,fullscreen",
+				plugins: "safari,fullscreen,paste",
 				theme: "advanced",
-				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 				theme_advanced_buttons2: "",
 				theme_advanced_buttons3: "",
 				extended_valid_elements: "+a[id|style|rel|rev|charset|hreflang|dir|lang|tabindex|accesskey|type|name|href|target:_blank|title|class|onfocus|onblur|onclick|" + 
@@ -908,10 +908,10 @@
 				mode: tinyMode,
 				editor_selector: "ambrosiaHtmlEdit_small",
 				convert_urls: false,
-				plugins: "safari,fullscreen",
+				plugins: "safari,fullscreen,paste",
 				theme: "advanced",
 				theme_advanced_buttons1: "undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist",
-				theme_advanced_buttons2: "fullscreen,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+				theme_advanced_buttons2: "fullscreen,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 				theme_advanced_buttons3: "",
 				extended_valid_elements: "+a[id|style|rel|rev|charset|hreflang|dir|lang|tabindex|accesskey|type|name|href|target:_blank|title|class|onfocus|onblur|onclick|" + 
 										 "ondblclick|onmousedown|onmouseup|onmouseover|onmousemove|onmouseout|onkeypress|onkeydown|onkeyup]",
@@ -919,7 +919,7 @@
 				fullscreen_settings :
 					{
 						theme_advanced_toolbar_location : "top",
-						theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+						theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 						theme_advanced_buttons2: ""
 					},
 				tab_focus: ":prev,:next",
@@ -940,9 +940,9 @@
 				mode: tinyMode,
 				editor_selector: "ambrosiaHtmlEdit_full",
 				convert_urls: false,
-				plugins: "safari,fullscreen",
+				plugins: "safari,fullscreen,paste",
 				theme: "advanced",
-				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 				theme_advanced_buttons2: "",
 				theme_advanced_buttons3: "",
 				file_browser_callback: "ambrosiaFileBrowser",
@@ -967,9 +967,9 @@
 				mode: tinyMode,
 				editor_selector: "ambrosiaHtmlEdit_tall",
 				convert_urls: false,
-				plugins: "safari,fullscreen",
+				plugins: "safari,fullscreen,paste",
 				theme: "advanced",
-				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+				theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 				theme_advanced_buttons2: "",
 				theme_advanced_buttons3: "",
 				file_browser_callback: "ambrosiaFileBrowser",
@@ -994,10 +994,10 @@
 				mode: tinyMode,
 				editor_selector: "ambrosiaHtmlEdit_small",
 				convert_urls: false,
-				plugins: "safari,fullscreen",
+				plugins: "safari,fullscreen,paste",
 				theme: "advanced",
 				theme_advanced_buttons1: "undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist",
-				theme_advanced_buttons2: "fullscreen,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code",
+				theme_advanced_buttons2: "fullscreen,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolor,backcolor,charmap,separator,code,pasteword",
 				theme_advanced_buttons3: "",
 				file_browser_callback: "ambrosiaFileBrowser",
 				extended_valid_elements: "+a[id|style|rel|rev|charset|hreflang|dir|lang|tabindex|accesskey|type|name|href|target:_blank|title|class|onfocus|onblur|onclick|" + 
@@ -1006,7 +1006,7 @@
 				fullscreen_settings :
 					{
 						theme_advanced_toolbar_location : "top",
-						theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolorpicker,backcolorpicker,charmap,separator,code",
+						theme_advanced_buttons1: "fullscreen,separator,undo,redo,separator,bold,italic,underline,strikethrough,sub,sup,separator,outdent,indent,justifyleft,justifycenter,justifyright,justifyfull,separator,bullist,numlist,separator,link,unlink,image,separator,fontselect,fontsizeselect,forecolorpicker,backcolorpicker,charmap,separator,code,pasteword",
 						theme_advanced_buttons2: ""
 					},
 				tab_focus: ":prev,:next",
